mysite/polls/admin.py

```diff
@@ -1,14 +1,61 @@
+from django import forms
 from django.contrib import admin
+# from django.contrib import messages
 
 from .models import Choice, Question
 
+import yaml
+
+
+def contains_curse_words(entry):
+    with open("polls/curse_words.yaml", 'r') as stream:
+        curse_words = yaml.safe_load(stream)
+    my_list = []
+    for curse_word in curse_words:
+        if curse_word in entry.lower():
+            my_list.append(curse_word)
+    return my_list
+
+# def validate_no_curse_words(field, entry, form):
+#     flag = True
+#     with open("polls/curse_words.yaml", 'r') as stream:
+#         curse_words = yaml.safe_load(stream)
+#     for curse_word in curse_words:
+#         if curse_word in entry.lower():
+#             msg = "Please don't use curse word: " + curse_word
+#             form.add_error(field, msg)
+#             flag = False
+#     return flag
+
+
+class QuestionAdminForm(forms.ModelForm):
+    def clean(self):
+        cleaned_data = super().clean()
+        field = "question_text"
+        entry = cleaned_data.get(field)
+        for curse_word in contains_curse_words(entry):
```

Comment:
I feel a little uncomfortable where an empty list being returned by the `contains_curse_words` function is implicitly interpreted by later logic. Could we do any refactor to make that more clear (e.g. docs, restructuring the method)? @aivu, I wanted to get your thoughts on this as well.

Reply:
I am open to suggestions. I have added a doctoring for now. If you think that is sufficient, you can mark this as resolved.